app/main/views/digital_services_framework.py

```diff
@@ -1,12 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from flask import render_template
-from ...main import main
-
-
-@main.route('/digital-services/framework')
```

Comment:
We should return a 410 (GONE) here I think, rather than deleting the route completely:
`The 410 (Gone) status code SHOULD be used if the server knows, through some internally configurable mechanism, that an old resource is permanently unavailable and has no forwarding address.`
https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html